node/src/blockchain/blockchain_bridge.rs

```
@@ -1021,20 +1056,31 @@ mod tests {
         subject
             .payment_confirmation
             //due to this None we would've panicked if we tried to send a msg
-            .report_transaction_receipts_sub_opt = None;
+            .report_transaction_receipts_sub_opt = Some(recipient);
         let msg = RequestTransactionReceipts {
             pending_payable: vec![fingerprint_1, fingerprint_2],
             response_skeleton_opt: None,
         };
+        let system = System::new("test");
 
         let _ = subject.handle_scan(
             BlockchainBridge::handle_request_transaction_receipts,
             ScanType::PendingPayables,
             &msg,
         );
 
+        System::current().stop();
+        system.run();
         let get_transaction_receipts_params = get_transaction_receipt_params_arc.lock().unwrap();
+        let recording = accountant_recording.lock().unwrap();
         assert_eq!(*get_transaction_receipts_params, vec![hash_1]);
+        assert_eq!(
+            recording.get_record::<ReportTransactionReceipts>(0),
+            &ReportTransactionReceipts {
+                fingerprints_with_receipts: vec![],
```

I was thinking of this awhile... Hmmm... Given all that what I've sen so far, I 'd be happier if this did NOT return just a vector but maybe a result of a vector? So that you can let Accountant know that the process not only found no tx receipts to be reported (the empty vector) but it also worked wrong, and you could report a certain error. 

I don't like that Accountant will behave the same in both situations by the time of the hand-over of the empty vector. But then I also think it wouldn't harm anything really if it logged an error here and sent the vector just empty.  By all views, it would somehow be nearest to the way we did it before...

Ok. I don't have an enough firm opinion on this. So take it as thought food and feel free to do nothing for this comment.